projects/storefrontapp/src/environments/environment.ts

```diff
@@ -8,11 +8,10 @@ import { Environment } from './models/environment.model';
 export const environment: Environment = {
   production: false,
   occBaseUrl:
-    build.process.env.SPARTACUS_BASE_URL ??
-    'https://spartacus-devci767.eastus.cloudapp.azure.com:9002',
-  // 'https://spartacus-dev0.eastus.cloudapp.azure.com:9002',
-  occApiPrefix: build.process.env.SPARTACUS_API_PREFIX ?? '/occ/v2/',
-  cds: build.process.env.SPARTACUS_CDS ?? false,
-  b2b: build.process.env.SPARTACUS_B2B ?? false,
-  cdc: build.process.env.SPARTACUS_CDC ?? false,
+    buildProcess.env.CX_BASE_URL ??
+    'https://spartacus-dev0.eastus.cloudapp.azure.com:9002',
+  occApiPrefix: '/occ/v2/',
```

Comment:
If we no longer need to test 1905 why not to inline it in the app?

Reply:
We will need 2011 soon.

Reply:
We will have different api prefix in 2011?

Reply:
ah, I see what you mean. Nope, we could inline in the app but I would keep it like this for this PR and just create a new ticket for that. What do you think?

Reply:
If you prefer new ticket, sure. Let's do this separately.